docs/getting-started/installation.md

```diff
@@ -5,143 +5,133 @@ icon: material/cogs
-
-=== "Ventoy"
-
-    1. Download and unzip [Ventoy](https://github.com/ventoy/Ventoy).
-    2. Plug in your USB Stick and open ``Ventoy2Disk.exe``.
-    2. Disable ``Secure Boot Support`` in the ``Option`` menu, if you have it disabled in your BIOS and you use ``UEFI``.
-    3. Then in the same tab, configure ``Partition Style`` depending on the BIOS Mode:
-        - If you are using ``UEFI``, select ``GPT``.
-        - If you are using ``Legacy``, select ``MBR``.
-    5. Click ``Install`` and then click OK in the two warnings.
-    6. Extract ``Atlas [version].zip``, to the root directory of your USB.
-        - Move AME Wizard to the ``Atlas`` folder on the USB Stick.
-        - If you downloaded drivers for your Ethernet/Wi-Fi adapter, copy them over.
-    7. Move the iso onto your USB Stick.
-
-=== "Rufus"
-
-    1. Download and open [Rufus](https://github.com/pbatard/rufus).
-    2. Plug in your USB Stick.
-    3. Click the ``SELECT`` button and pick your ISO file.
-    4. Configure ``Partition scheme``
-        - If you are using ``UEFI``, select ``GPT``.
-        - If you are using ``Legacy``, select``MBR``.
-    4. Click ``START`` and in the ``Windows User Experience`` wizard, and then click ``OK``.
-    5. Press ``OK`` again and your USB will be erased and turned into a bootable USB.
-    6. Extract ``Atlas [version].zip``, to the root directory of your USB.
-        - Additionally, move AME Wizard to the ``Atlas [version]`` folder.
-        - If you downloaded drivers for your Ethernet/Wi-Fi adapter, copy them over.
-
-## :material-power: Booting into your USB
-
-1. Disconnect the internet cable from your device and do not connect to the internet during the installation.
-    - This stops the Windows OOBE from making unauthorized changes that could later hurt the Atlas installation process.
-2. Boot into your USB using Boot Menu/BIOS.
-    - If you are using Ventoy, choose the ISO file that you moved to the USB Stick in the program menu.
-4. Proceed with the Windows installation as usual.
-    - If asked, choose your desired edition (Professional is recommended).
-5. Finish installing Windows in the OOBE, and follow the video below:
-
-    <iframe src="https://streamable.com/e/21wdvv?loop=0" width="560" height="420" frameborder="0" allowfullscreen></iframe>
-    
-    If the video above does not work, use the video provided here:
-    
-    - [Windows 10+ OOBE](https://raw.githubusercontent.com/amitxv/PC-Tuning/main/media/oobe-windows10+-example.mp4)
+You should now have an Atlas folder with all the files you need on the root directory of your external drive.
+
+### :material-harddisk-plus: Creating a temporary partition
+
+This is for the case that you have nothing to put the Atlas files on, which won't be wiped by reinstalling Windows. If you have already done the [Preparing Atlas](/getting-started/installation/#preparing-atlas) steps, skip to [Reinstalling Windows](/getting-started/installation/#reinstalling-windows).
```

Comment:
Bold the first sentence for the sake of clarity for why this may be optional?

Reply:
I'll put it as an admonition

Reply:
Added as an admonition